test/datadog/values/process-run-in-core-envvars.yaml

```diff
@@ -0,0 +1,10 @@
+agents:
```

Comment:
Added this because I could not figure out how to add an envvar override [here](https://github.com/DataDog/helm-charts/blob/c9d89c4a7897f7d969afac382e19e46172e012db/test/datadog/process_agent_test.go#L240). Attempts to represent as a string resulted in render failures :/